pkg/apis/policy/v1beta1/doc.go

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Please do NOT do this in future - this commit is super huge and it contains both autogenerated code as well as manually writte - this is unreviable.